spec/features/admin/index_spec.rb

```diff
@@ -0,0 +1,80 @@
+     
+      order1 = user.orders.create!(name: 'Steve Meyers', address: "123 Main St.", city: "Fort Collins", state: "GA", zip: "66666", status: 0, user: user)     
+      order1.item_orders.create!(item: pull_toy, order: order1, price: pull_toy.price, quantity: 7)
+     
+      order2 = user.orders.create!(name: 'Steve Meyers 2', address: "123 Main St.", city: "Fort Collins", state: "GA", zip: "66666", status: 1, user: user)     
+      order2.item_orders.create!(item: dog_bone, order: order2, price: dog_bone.price, quantity: 4)
+     
+      order3 = user.orders.create!(name: 'Steve Meyers 3', address: "123 Main St.", city: "Fort Collins", state: "GA", zip: "66666", status: 2, user: user)     
+      order3.item_orders.create!(item: dog_ball, order: order3, price: dog_ball.price, quantity: 3)
+     
+      order4 = user.orders.create!(name: 'Steve Meyers 4', address: "123 Main St.", city: "Fort Collins", state: "GA", zip: "66666", status: 3, user: user)     
+      order4.item_orders.create!(item: dog_bowl, order: order4, price: dog_bowl.price, quantity: 2)
+
+      visit admin_dashboard_path
+
+      expect(page).to have_content("Admin Dashboard")
+      within "#order-#{order2.id}" do
+        expect(page).to have_content(order2.id)
+        expect(page).to have_content(order2.created_at.to_date)
+        expect(page).to have_content(order2.status)
+        expect(page).to have_link(order2.name)
+        click_on order2.name 
+        expect(current_path).to eq("/admin/users/#{order2.user.id}")
+      end
+
+      visit admin_dashboard_path
+      
+      within "#order-#{order1.id}" do 
+        expect(page).to have_link(order1.name)
+        expect(page).to have_content(order1.id)
+        expect(page).to have_content(order1.created_at.to_date)
+        expect(page).to have_content(order1.status)
+      end
+
+      within "#order-#{order3.id}" do 
+        expect(page).to have_link(order3.name)
+        expect(page).to have_content(order3.id)
+        expect(page).to have_content(order3.created_at.to_date)
+        expect(page).to have_content(order3.status)
+      end
+
+      within "#order-#{order4.id}" do 
+        expect(page).to have_link(order4.name)
+        expect(page).to have_content(order4.id)
+        expect(page).to have_content(order4.created_at.to_date)
+        expect(page).to have_content(order4.status)
+      end
+      save_and_open_page
+    end 
+  end 
+end 
+
+# As an admin user
+# When I visit my admin dashboard ("/admin")
+# Then I see all orders in the system.
+# For each order I see the following information:
+
+# user who placed the order, which links to admin view of user profile
+# order id
+# date the order was created
+# Orders are sorted by "status" in this order:
+
+# packaged
+# pending
+# shipped
+# cancelled
```

Comment:
👀

Reply:
good catch, had a save and open page in there too!